Basic-DSA/GenericLinkedListImpl.java

```diff
@@ -0,0 +1,150 @@
+class Node<T> {
+    T data;
+    Node<T> next;
+    Node(T data) {
+        this.data = data;
+        this.next = null;
+    }
+}
+
+interface GenericList<T> {
```

Comment:
can you move this interface to seperate file?

Reply:
ok